From b936c12a25851d2e130df91fe22a8daa898ff580 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Tue, 2 Feb 2016 09:48:33 +0100
Subject: [PATCH 10/12] s2: Add LED blink at boot up

The main leds are flashing white while the board boots up

Ticket #872
---
 arch/arm/boot/dts/imx6sx-s2.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-s2.dts b/arch/arm/boot/dts/imx6sx-s2.dts
index e99734b..778b4bd 100644
--- a/arch/arm/boot/dts/imx6sx-s2.dts
+++ b/arch/arm/boot/dts/imx6sx-s2.dts
@@ -158,17 +158,20 @@
 			chan-name = "R1";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 		chan1 {
 			chan-name = "G1";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 
 		chan2 {
 			chan-name = "B1";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 	};
 
@@ -182,17 +185,20 @@
 			chan-name = "R2";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 		chan1 {
 			chan-name = "G2";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 
 		chan2 {
 			chan-name = "B2";
 			led-cur = /bits/ 8 <0x20>;
 			max-cur = /bits/ 8 <0x60>;
+            linux,default-trigger = "heartbeat";
 		};
 	};
 };
-- 
1.9.1

